app/Http/Controllers/ClubEventController.php

```diff
@@ -296,13 +267,14 @@ public function edit($id)
 
 
         // get a list of available job types
-        $jobtypes = Jobtype::where('jbtyp_is_archived', '=', '0')
+        $jobtypes = ShiftType::where('jbtyp_is_archived', '=', '0')
```

Comment:
Yes, after this PR is merged we should also rename the other models (each in a separate branch).